packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -384,6 +393,11 @@ export abstract class FluidDataStoreContext
 						value: this.id,
 						tag: TelemetryDataTag.CodeArtifact,
 					},
+					packageName: {
+						value: JSON.stringify(this.pkg),
```

Comment:
do we stringify because we do not have support for arrays?

Reply:
yes

Reply:
FYI some pre-requisite work to support arrays here has happened.  But not done.